robot/rospackages/src/camera_vision/scripts/ar_tracker.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python2
+import sys
+
+import rospy
+import cv2
+from sensor_msgs.msg import Image, CameraInfo
+from ar_track_alvar_msgs.msg import AlvarMarkers
+from cv_bridge import CvBridge, CvBridgeError
+import numpy as np
+
+class arTrackerDemo():
```

Comment:
Does it need to be called Demo

Reply:
No it does not. Changed it to `arTracker`